module4/lessons/Week1/STARInterviewTechnique.md

```diff
@@ -0,0 +1,114 @@
+* What is one of your strengths? How have you seen this strength in practice as a developer? 
+
+Use this as an opportunity to tell stories about your relevant experience and highlight your strengths. A great template to use to plan out your stories is the STAR method:
+
+* **Situation**: The interviewer wants you to present a recent challenge and situation in which you found yourself.
+* **Task**: What were you required to achieve? The interviewer will be looking to see what you were trying to achieve from the situation. Some performance development methods use “Target” rather than “Task”. Job interview candidates who describe a “Target” they set themselves instead of an externally imposed “Task” emphasize their own intrinsic motivation to perform and to develop their performance.
+* **Action**: What did you do? The interviewer will be looking for information on what you did, why you did it, and what the alternatives were.
+* **Results**: What was the outcome of your actions? What did you achieve through your actions and did you meet your objectives? What did you learn from this experience and have you used this learning since?
+
+Example:
+
+*Tell me about a successful project that you’ve worked on.*
+
+In this example, focus on establishing the situation or scenario for the project. Set the stage for the listener – how many people were involved in this project? What was the timeline? What were the main technologies used?
+
+Then move into the task/target and discuss the objective of the project. What was the purpose? Who was the user?
+
+Then, walk through all of the actionable steps you/your team took to meet the objective of the project. What successes did you have? What challenges did you run into? What steps did you take to overcome those challenges?
+
+And finally, what were the results of this project? What were you able to accomplish? What would you have done differently?
+
+### Practicing STAR
+
+Time to practice the STAR technique. You and your partner will take turns playing the role of interviewer and candidate, and you will both answer the following question. 
+
+*What is one of your strengths? How have you seen this strength in practice as a developer?*
+
+When you are not answering the question, listen closely to the person who is speaking so you can share feedback:
+* Did they include all four parts of the STAR technique?
+* Did they speak in specifics (ex: “I did XYZ”) or just in hypotheticals (ex: “I would do XYZ”)?
+* Did they provide enough context to help you understand their story?
+
+Once you have both had a chance to answer, give feedback and discuss how you felt that went.
+
+<aside class="instructor-notes" markdown="1">
+<p><strong>Instructor Note</strong><br>Because this is the student's first time trying STAR I would give them 2 minutes to think about what story they want to tell before putting them in breakout rooms.</p>

```

Comment:
I think this is a great call-out.  We could even make this part of the activity outline.